### DISCUSSION TITLE
Idea on how to Zero the tool

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#218] [Ideas]
Ive been having a think about an accurate way to zero the tool instead of having to take lots of cuts.

My idea is you get a dial indicator and mount it on the 3d Printed cross slide (modifications needed).

Then what you do is take a test cut to calibrate the tool as normal. Then what you do is move the tool backwards toward the dial indicator. The dial indicator would touch the toolpost (QCTP in my case). You would advance the cross slide towards it until it reads lets say 1mm.

Then you look at what the X value is on the controller, lets say its X-45.5.

Now you have your zero point.

As long as you dont move the dial indicator and keep it mounted there whenever you want to zero the lathe on the X, you can simple move until the dial indicator shows 1mm, then on the controller tell it its at X-45.5

This should be very repeatable assuming nothing moves, and it saves doing a test it whenever you turn on the lathe.

So in the case of the image below I need to advance the cross slide another 0.09mm towards the dial indicator. Then I can tell the controller im at X-45.5. Again this would need to be rigidly mounted and not moved. I use Polycarbonate filament for the cross slide motor and its very stiff so shouldnt have issues holding a dial indicator.

![IMG_9204](https://github.com/user-attachments/assets/9eb4b084-a0ea-410a-bc92-d448ec2488ad)

**Comment by wael007-hub** [comment]
Hello ,
i am interested in dial indicator and i am trying to make program to show indicator data result in graphical way .If you have any start for it .
Thank you and i appreciate your support in advance
